ivy/functional/backends/numpy/searching.py

```diff
@@ -55,6 +55,32 @@ def argmin(
     return np.array(ret, dtype=np.int64)
 
 
+def argpartition(
+    x: np.ndarray,
+    kth: Union[int, list, np.ndarray],
+    /,
+    *,
+    axis: Optional[Union[int, None]] = -1,
+    kind: Optional[str] = 'introselect',
+    order: Optional[Union[str, list]] = None,
+    output_dtype: Optional[Union[ivy.Dtype, ivy.NativeDtype]] = None
+) -> np.ndarray:
+    # Check if the given order one of the field else return pass order as None
+    try:
+        x[order]
+    except:
+        order = None
+
+    ret = np.argpartition(x, kth, axis=axis, kind=kind, order=order)
+    if output_dtype is not None:
+        output_dtype = ivy.as_native_dtype(output_dtype)
+        if output_dtype not in (np.int32, np.int64):
+            return ret.astype(dtype=np.int64)
+        else:
+            return ret.astype(dtype=output_dtype)
+    return ret.astype(dtype=np.int64)
+
+    
 argmin.support_native_out = True
```

Comment:
By the way, you should keep this line right after the argmin function and also add a similar line for argpartition stating that it does not support native out